.github/workflows/docs_deploy.yml

```diff
@@ -11,11 +11,11 @@ concurrency: "deploy-to-prod"
 jobs:
   chainloop_init:
     name: Chainloop Init
-    uses: chainloop-dev/labs/.github/workflows/chainloop_init.yml@7f4de29435dc009326587051f507d2cd8c77d28b
+    uses: chainloop-dev/labs/.github/workflows/chainloop_init.yml@d0298b552d20d018e4ec39a28661d225bab40057
     secrets:
       api_token: ${{ secrets.CHAINLOOP_ROBOT_ACCOUNT_DOCS_RELEASE }}
     with:
-      chainloop_labs_branch: 7f4de29435dc009326587051f507d2cd8c77d28b
+      chainloop_labs_branch: d0298b552d20d018e4ec39a28661d225bab40057
```

Comment:
do we know why we need to maintain a branch here? cc/ @danlishka

Reply:
I need to know which version of labs you want to install. In theibility was impo past, this flexrtant to me when building functions that work across GitLab, GitHub, and Azure DevOps.

Reply:
wouldn't pinging the reusable workflow version be enough?